src/main/java/com/dynxsty/dih4jda/interactions/commands/CommandRequirements.java

```diff
@@ -37,6 +39,15 @@ public final void requireUsers(Long... users) {
 		requiredUsers = Arrays.stream(users).collect(Collectors.toSet());
 	}
 
+	/**
+	 * Allows to require a set of {@link Long}s (user Ids) which are able to execute the corresponding command.
```

Comment:
```suggestion
	 * Defines a set of User-IDs as {@link Long}s which are able to execute the corresponding command.
```
That could confuse some people as it sounds like all roles would be required to execute the corresponding command. Besides that, it doesn't allow to do something - it does it. When you use that method it's not optional to "allow" these users